vdirsyncer/storage/dav.py

```diff
@@ -367,7 +367,7 @@ def init_and_remaining_args(cls, **kwargs):
 
     def __init__(self, url, username='', password='', verify=True, auth=None,
                  useragent=USERAGENT, verify_fingerprint=None,
-                 auth_cert=None):
+                 auth_cert=None, ignore_missing_href=False):
```

Comment:
Don't you need to somehow pass this data to this function?

I guess it should be a setting for the storage, is that what you had a mind?

If so, that would be an acceptable change.

Reply:
Hi @WhyNotHugo, yes correct. I have this in my config:
[storage example]
...
ignore_missing_href = true

I'm not sure about the name of the option. It probably won't be obvious what it does for someone that hasn't read the code